src/validate.js

```diff
@@ -95,6 +95,11 @@ function createErrorHandler(formData) {
       return {...acc, [key]: createErrorHandler(formData[key])};
     }, handler);
   }
+  if(Array.isArray(formData)){
+    return Object.assign([],formData.reduce((acc, item, index) => {
```

Comment:
Ohh, I see what you did here, smart; we need keys as object properties. That should actually work.